# goal
string package_name
string colour
geometry_msgs/Pose package_pose
geometry_msgs/Pose ur5_pose
---
# result
bool success
---
# feedback
geometry_msgs/Pose arm_pose